cmake/Packaging.cmake

```diff
@@ -0,0 +1,116 @@
+#-----------------------------------------------
+# packaging
+#-----------------------------------------------
+
+option(ENABLE_PACKAGING "Enable Packaging" ON)
+
+if (CMAKE_C_COMPILER MATCHES gcc)
+    exec_program(${CMAKE_C_COMPILER}
+        ARGS -dumpversion
+        OUTPUT_VARIABLE PACKAGE_COMPILER)
+    set(PACKAGE_COMPILER gcc-${PACKAGE_COMPILER})
+else()
+    set(PACKAGE_COMPILER ${CMAKE_CXX_COMPILER})
+endif()
+
+if (UNIX AND NOT APPLE) # LINUX
+    option(BUILD_RPM "Enable RPM Packaging" OFF)
+endif()
+
+#   Do packaging only if there is no other (upward) packaging already being done.
+
+if (NOT PACKAGE_NAME)
+    if (ENABLE_PACKAGING OR BUILD_RPM)
+        include(InstallRequiredSystemLibraries)
+
+        configure_file(${MATIO_SOURCE_DIR}/README    README      COPYONLY)
+        configure_file(${MATIO_SOURCE_DIR}/README.md README.md   COPYONLY)
+        configure_file(${MATIO_SOURCE_DIR}/COPYING   COPYING.txt COPYONLY)
+
+        set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "matio library for matlab IOs")
+        set(CPACK_PACKAGE_VENDOR "Christopher Hulbert")
+        set(CPACK_PACKAGE_DESCRIPTION_FILE "${MATIO_BINARY_DIR}/README.md")
+        set(CPACK_RESOURCE_FILE_LICENSE "${MATIO_BINARY_DIR}/COPYING.txt")
+        set(CPACK_PACKAGE_INSTALL_DIRECTORY "matio")
+        set(CPACK_PACKAGE_CONTACT "cch@isl-inc.com")
```

Comment:
Better use the sf mail address

Reply:
Done, I have put the two sf addresses I found in README.md. 
In the long run a mailist might be better....